maven-scm-providers/maven-scm-providers-git/maven-scm-provider-jgit/pom.xml

```diff
@@ -47,13 +47,12 @@
     <dependency>
       <groupId>org.eclipse.jgit</groupId>
       <artifactId>org.eclipse.jgit</artifactId>
-      <version>4.5.4.201711221230-r</version>
-      <exclusions>
-        <exclusion>
-          <groupId>commons-logging</groupId>
-          <artifactId>commons-logging</artifactId>
-        </exclusion>
-      </exclusions>
+      <version>5.13.1.202206130422-r</version><!-- since version 6 requires Java 11 -->
+    </dependency>
+    <dependency>
+      <groupId>org.eclipse.jgit</groupId>
+      <artifactId>org.eclipse.jgit.ssh.apache</artifactId>
+      <version>5.13.1.202206130422-r</version><!-- since version 6 requires Java 11 -->
```

Comment:
Move this to a property along with the comment.

Reply:
Done in https://github.com/apache/maven-scm/pull/153/commits/14e6517b77d5bb9046e9945c4f5d45555ad559c7.